<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PassportExtendedFormControl</class>
 <widget class="QWidget" name="PassportExtendedFormControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="sourceLabel">
       <property name="text">
        <string>Data source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sourceLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="altitudeLabel">
       <property name="text">
        <string>Altitude</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="altitudeComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="intervalLabel">
       <property name="text">
        <string>Perforation interval</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="perforationIntervalSpinBox">
       <property name="minimum">
        <double>-10000.000000000000000</double>
       </property>
       <property name="maximum">
        <double>50000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="stratumLabel">
       <property name="text">
        <string>Stratum/Suite</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="stratumLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="fluidTypeLabel">
       <property name="text">
        <string>Fluid type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fluidTypeLineEdit"/>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="pumpLabel">
       <property name="text">
        <string>Booster pumping station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pumpComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="clusterLabel">
       <property name="text">
        <string>Cluster pumping station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="clusterComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="oilAndGasProdDepLabel">
       <property name="text">
        <string>Oil and gas production department</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="oilAndGasProdDepComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>sourceLineEdit</tabstop>
  <tabstop>altitudeComboBox</tabstop>
  <tabstop>perforationIntervalSpinBox</tabstop>
  <tabstop>stratumLineEdit</tabstop>
  <tabstop>fluidTypeLineEdit</tabstop>
  <tabstop>pumpComboBox</tabstop>
  <tabstop>clusterComboBox</tabstop>
  <tabstop>oilAndGasProdDepComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
